src/components/PaginationButton.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+  <button
+    :class="[
+      'pagination-button',
+      ...($slots.default ? [`pagination-button--${direction}`] : []),
```

Comment:
```suggestion
      $slots.default ? [`pagination-button--${direction}`] : [],
```

Seems like this will work the same